### PR TITLE
Use RunScript for python actions

### PR DESCRIPTION
**Pull request by McSwindler**
I was having issues with python actions minimizing the Kodi window, I changed it to use xbmc.runscript.

I also tweaked the actions script to be able to read relative paths in action files, it should work fine on unix and windows, however I only tested on windows.

### CODE REVIEW

**Comment by ruuk** [comment]
Thanks for the PR

A few issues.
1. What you've done in with the python/script action duplicates the addon action :)
2. The python/script action is specifically meant to run python scripts in the OS python environment rather than in the Kodi Python environment
3. It looks like _absolutizeCommand() makes paths relative to the parent directory of the action.py file and also doesn't account for absolute paths

Also, I've believe I've fixed the issue with Kodi being minimized for python and command actions. The python action also relies on python being in the system path, so you also may have initially had an issue with that as well. If you don't have python in the system path, you can just also just use the command action and run Python directly giving it's absolute path and passing the python script path as an argument.

**Comment by McSwindler** [comment]
I completely missed the addon action...I feel kind of dumb now, I'll try out the minimization fix later and let you know how it works out. I remember having to add python to my path since I was getting a different error when it wasn't.

However, on the absolutize part, I am setting self.path to the cvaction file path, so it should be getting the parent dir of that file. And os.path.join handles absolute paths appropriately, here are some examples:

Windows
```
>>> import os
>>> os.path.normpath(os.path.join(os.path.dirname("C:\\Users\\James\\action.cvaction"), "C:\\Test\\do-something.py"))
'C:\\Test\\do-something.py'
>>> os.path.normpath(os.path.join(os.path.dirname("C:\\Users\\James\\action.cvaction"), "pie\\apple\\do-something.py"))
'C:\\Users\\James\\pie\\apple\\do-something.py'
>>> os.path.normpath(os.path.join(os.path.dirname("C:\\Users\\James\\action.cvaction"), "..\\pie\\apple\\do-something.py"))
'C:\\Users\\pie\\apple\\do-something.py'
```

Linux
```
>>> import os
>>> os.path.normpath(os.path.join(os.path.dirname("/home/james/action.cvaction"), "/pie/apple/do-something.py"))
'/pie/apple/do-something.py'
>>> os.path.normpath(os.path.join(os.path.dirname("/home/james/action.cvaction"), "pie/apple/do-something.py"))
'/home/james/pie/apple/do-something.py'
>>> os.path.normpath(os.path.join(os.path.dirname("/home/james/action.cvaction"), "../pie/apple/do-something.py"))
'/home/pie/apple/do-something.py'
```

**Comment by ruuk** [comment]
For some reason I was looking at self.path and thinking of __file__. I was apparently temporarily insane :)

**Comment by McSwindler** [comment]
The change you made works great, just forgot the self param on SubprocessActionCommand.getStartupInfo()

**Comment by ruuk** [comment]
Thanks for testing and finding that. I copy/pasted the function from some of my other code :)

File: lib/cinemavision/actions.py [hunk 1]
```diff
@@ -99,10 +107,9 @@ def getStartupInfo():
 class ScriptCommand(SubprocessActionCommand):
     type = 'SCRIPT'
 
-    def execute(self):
-        command = ['python', self.commandData]
+    def execute(self):        
+        command = [self._absolutizeCommand()]
```
Comment (ruuk) [inline lib/cinemavision/actions.py]: I think this needs to be changed to `command = ['python', self._absolutizeCommand()]` and then I can merge this.

**Comment by McSwindler** [comment]
oops, should be good now :)

**Pull request merged**